lib/lib-table-cluster/src/main/java/org/sagebionetworks/table/cluster/TableIndexDAOImpl.java

```diff
@@ -134,35 +135,30 @@ public boolean deleteTable(String tableId) {
 	}
 
 	@Override
-	public void createOrUpdateOrDeleteRows(final RowSet rowset,
-			final List<ColumnModel> schema) {
-		if (rowset == null)
-			throw new IllegalArgumentException("Rowset cannot be null");
-		if (schema == null)
-			throw new IllegalArgumentException("Current schema cannot be null");
-
+	public void createOrUpdateOrDeleteRows(final Grouping grouping) {

```

Comment:
Still need to pass the schema in so we can take the intersection of the grouping and the schema when generating the SQL.